downloads/swan-lake-release-notes/swan-lake-alpha/RELEASE_NOTE.md

```diff
@@ -0,0 +1,1562 @@
+    - [Developer Tools](#developer-tools)
+        - [Language Server](#language-server)
+        - [Project API](#project-api)
+        - [Debugger](#debugger)
+        - [Test Framework](#test-framework)
+        - [Bindgen Tool](#bindgen-tool)
+        - [Maven Resolver](#maven-resolver)
+        - [Ballerina Shell REPL [EXPERIMENTAL]](#ballerina-shell-repl-experimental)
+        - [Documentation](#documentation)
+    - [Standard Library](#standard-library)
+        - [HTTP Module Improvements](#http-module-improvements)
+        - [WebSocket Module Improvements](#websocket-module-improvements)
+        - [gRPC Module Improvements](#grpc-module-improvements)
+        - [Security Improvements](#security-improvements)
+        - [GraphQL Module Improvements](#graphql-module-improvements)
+        - [Common Changes in Messaging Modules](#common-changes-in-messaging-modules)
+        - [Kafka Module Improvements](#kafka-module-improvements)
+        - [NATS Module Improvements](#nats-module-improvements)
+        - [NATS Streaming Module Improvements](#nats-streaming-module-improvements)
+        - [RabbitMQ Module Improvements](#rabbitmq-module-improvements)
+        - [Time Module Improvements](#time-module-improvements)
+        - [Rename System Module to OS](#rename-system-module-to-os)
+        - [Runtime Module Improvements](#runtime-module-improvements)
+        - [Email Module Improvements](#email-module-improvements)
+        - [WebSub Module Improvements](#websub-module-improvements)
+        - [Introduced New Modules](#introduced-new-modules)
+        - [Removed Modules](#removed-modules)
+    - [Code to Cloud](#code-to-cloud)
+    - [Observability](#observability)
+    - [Breaking Changes](#breaking-changes)
+        - [Language](#language)
+    - [Taint Analyzer Update](#taint-analyzer-update)
+
+#### Updating Ballerina
+
+You can use the update tool to update to Ballerina Swan Lake Alpha as follows.
+
+##### For Existing Users
+
+If you are already using Ballerina, you can directly update your distribution to the Swan Lake channel using the [Ballerina Update Tool](/swan-lake/learn/keeping-ballerina-up-to-date/). To do this, first, execute the command below to get the update tool updated to its latest version. 
+                        
+> `bal update`
+
+ Next, execute the command below to update to Swan Lake Alpha.
+
+ > `bal dist pull alpha`                 
+
+However, if you are using a Ballerina version below 1.1.0, install via the [installers](/downloads/#swanlake).
+
+##### For New Users
+
+If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
+
+#### Highlights
+
+- The `ballerina` command is renamed to `bal`
+    Now onwards, all the commands will start with `bal` E.g., `bal -v`, `bal build`, `bal dist list`.
+- Introduction of hierarchical package names
+    Now, the package name can take the form of `package-name := identifier(.identifer)*`.
```

Comment:
Looks like this line break is not visible in the MD view. They appear in the same line without a period.

Reply:
Do we need multi sentence descriptions for this section anyways?

Reply:
We can go with single line sentence.

Reply:
Fixed.